lightbulb/commands/base.py

```diff
@@ -172,6 +172,18 @@ class OptionLike:
     """
     autocomplete: bool = False
     """Whether the option should be autocompleted or not. This only affects slash commands."""
+    name_localizations: t.Mapping[t.Union[hikari.Locale, str], str] = dataclasses.field(default_factory=dict)
+    """
+    A set of name localizations for this option
+    
+    .. versionadded:: 2.2.5
```

Comment:
Also, these versionadded would need a bump

Reply:
Fixed in c5888c1b1d9e40f58b7d092bb3f5aa454968c7e7.